### PR TITLE
Dapr CLI version update

### PR DESCRIPTION
**Pull request by mml5bg**
## Describe your changes

Dapr CLI version update to version where https://github.com/dapr/cli/issues/1043 is fixed.

## Issue ticket number and link

[AB#11317](https://github.com/dapr/cli/issues/1043)

## Checklist - Manual tasks

* [x] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

**Comment by dennismeister93** [comment]
@mml5bg please also upgrade the version in all the AppManifests of the example apps! Thanks!

File: setup.cfg [hunk 1]
```diff
@@ -5,7 +5,7 @@ exclude =
 max-line-length = 88
 max-complexity = 18
 select = B,C,E,F,W,T4,B9,B950
-ignore = W503, B903, E402
+ignore = W503, B903, E402, B905
```
Comment (dennismeister93) [inline setup.cfg:8]: How does this belong together with the dapr cli?

**Comment by mml5bg** [comment]
> How does this belong together with the dapr cli? 

Actually, it doesn't belong to the dapr cli, but somehow that error was discovered from the linter, could be historical thing, but anyhow, we are going to change linter settings in other PR, it will come soon.

**Comment by mml5bg** [comment]
> @mml5bg please also upgrade the version in all the AppManifests of the example apps! Thanks! 

Done!

**Comment by dennismeister93** [comment]
> > How does this belong together with the dapr cli?
> 
> Actually, it doesn't belong to the dapr cli, but somehow that error was discovered from the linter, could be historical thing, but anyhow, we are going to change linter settings in other PR, it will come soon.

Ok this change is now in anyways after the last SDK PR #43 

Thx! LGTM

**Review by dennismeister93 — APPROVED** [review]

**Pull request merged**